src/ai/backend/manager/api/vfolder.py

```diff
@@ -1805,6 +1805,30 @@ async def delete(request: web.Request) -> web.Response:
     return web.Response(status=204)
 
 
+async def leave_request(request: web.Request, params: Any) -> web.Response:
```

Comment:
Why is this function named "request"? It just deletes the permission entry "blindly". If this is an internal database model manipulation, this should not take `web.Request` for the principle of separation of concerns, and should moved to `ai.backend.manager.models.vfolder`.